modules/email_attestation.js

```diff
@@ -4,12 +4,28 @@ const conf = require('byteballcore/conf');
 const objectHash = require('byteballcore/object_hash.js');
 const db = require('byteballcore/db');
 const notifications = require('./notifications');
-const texts = require('./texts');
+const i18nModule = require("i18n");
+const arrWhitelistEmails = Object.keys(conf.objRewardWhiteListEmails);
+
+var arrLanguages = [];
+if (conf.isMultiLingual) {
+	for (var index in conf.languagesAvailable) {
+		arrLanguages.push(conf.languagesAvailable[index].file);
+	}
+}
+
+i18nModule.configure({
+	locales: arrLanguages,
+	directory: __dirname + '/../locales'
+});
+
+var i18n = {};
+i18nModule.init(i18n);
 
 function retryPostingAttestations() {
 	db.query(
-		`SELECT 
```

Comment:
seems like your editor is doing auto-edits without your knowledge, which makes the diffs too noisy. Please check the settings (no need to revert the edits)

Reply:
I am aware of this, i turned it off now, but IMHO trailing spaces doesn't make sense.

Reply:
When comparing changes, there is "Diff settings" in the header of the page on Github, which has option "Hide whitespace changes".